src/main/java/org/springframework/samples/petclinic/visit/VisitReactiveServices.java

```diff
@@ -91,4 +98,20 @@ public Mono<Void> deleteVisitById(UUID visitId) {
                    .map(visitDao::delete)
                    .then();
     }
+
+    /**
+     * Spring interface {@link InitializingBean} let you execute some
+     * code after bean has been initialized.
+     *
+     * Here we enter default values for Pets and Pet visits.
+     */
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        UUID petUuid = UUID.fromString("11111111-1111-1111-1111-111111111111");
+        Mono.from(petDao.upsert(new PetEntity(UUID.randomUUID(), petUuid,
+                "Dog", "Ollie", LocalDate.of(2021, 5, 10)))).subscribe();
+        for (int i = 0; i < 10; i++)
+            Mono.from(visitDao.upsert(new VisitEntity(petUuid, UUID.randomUUID(),
+                    "General check up for Ollie", LocalDate.of(2023, 1, 25)))).subscribe();
```

Comment:
This was the change that allows the load generator container to receive data when it makes requests to `http://petclinic-backend:9966/petclinic/api/pets`.